django/santropolFeast/member/models.py

```diff
@@ -36,11 +36,11 @@ class Meta:
     gender = models.CharField(
         max_length=1,
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
+            ('F', 'Female'),
+            ('U', 'Unknown'),
         ),
-        default=1
+        default=2
```

Comment:
@maximest-pierre  Check out the Meal Model, the same thing has already been implemented: https://github.com/savoirfairelinux/santropol-feast/blob/dev/django/santropolFeast/meal/models.py#L23